applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -30,7 +30,12 @@ def __init__(self, Model, settings ):
         settings.ValidateAndAssignDefaults(default_parameters)
 
         self.body_model_part = Model[settings["model_part_name"].GetString()]
-        self.far_field_model_part = Model[settings["far_field_model_part_name"].GetString()]
+        far_field_model_part_name = settings["far_field_model_part_name"].GetString()
+        if far_field_model_part_name == "":
+            err_msg = "Empty model_part_name in ComputeLiftProcess\n"
+            err_msg += "Please specify the model part that contains the far field surface nodes"
+            raise Exception(err_msg)
```

Comment:
What if instead of throwing an error we just dont compute the FarField lift?

Reply:
sounds good!